Dockerfile

```diff
@@ -44,39 +42,33 @@ create, run, maintain and access a PostgreSQL DBMS server."
 LABEL summary="$SUMMARY" \
       description="$DESCRIPTION" \
       io.k8s.description="$DESCRIPTION" \
-      io.k8s.display-name="PostgreSQL 10" \
+      io.k8s.display-name="PostgreSQL 13" \
       io.openshift.expose-services="5432:postgresql" \
-      io.openshift.tags="database,postgresql,postgresql10,postgresql-10" \
+      io.openshift.tags="database,postgresql,postgresql13,postgresql-13" \
       io.openshift.s2i.assemble-user="26" \
-      name="rhel8/postgresql-10" \
-      com.redhat.component="postgresql-10-container" \
+      name="rhel9/postgresql-13" \
+      com.redhat.component="postgresql-13-container" \
       version="1" \
       com.redhat.license_terms="https://www.redhat.com/en/about/red-hat-end-user-license-agreements#rhel" \
-      usage="podman run -d --name postgresql_database -e POSTGRESQL_USER=user -e POSTGRESQL_PASSWORD=pass -e POSTGRESQL_DATABASE=db -p 5432:5432 rhel8/postgresql-10" \
+      usage="podman run -d --name postgresql_database -e POSTGRESQL_USER=user -e POSTGRESQL_PASSWORD=pass -e POSTGRESQL_DATABASE=db -p 5432:5432 rhel9/postgresql-13" \
       maintainer="SoftwareCollections.org <sclorg@redhat.com>"
 
 EXPOSE 5432
 
-COPY --from=postgresql_container_source /postgresql-container/10/root/usr/libexec/fix-permissions /usr/libexec/fix-permissions
+COPY --from=postgresql_container_source /postgresql-container/13/root/usr/libexec/fix-permissions /usr/libexec/fix-permissions
 
 # This image must forever use UID 26 for postgres user so our volumes are
 # safe in the future. This should *never* change, the last test is there
 # to make sure of that.
-RUN if [ "$(uname -m)" != "s390x" ]; then \
-      yum -y --setopt=tsflags=nodocs install \
-         http://mirror.centos.org/centos/8-stream/BaseOS/${ARCH}/os/Packages/centos-stream-repos-8-2.el8.noarch.rpm \
-         http://mirror.centos.org/centos/8-stream/BaseOS/${ARCH}/os/Packages/centos-gpg-keys-8-2.el8.noarch.rpm && \
-      yum -y module enable postgresql:10 && \
```

Comment:
Are we no longer using yum modules because pg 13 is the default now?

I'll have to run through all of my prior postgresql 13 changes and test again to see if that changes anything relevant.

Reply:
I think it should be ok if it uses default paths and not versioned paths.

Reply:
I think going to ubi9 will allow us to close/remove #41 as much of the changes are the same.

Reply:
Yes, PG 13 is the default for UBI9, so there is no PG13 module here.